papers/bradley_dice/55_bradley_dice.rst

```diff
@@ -0,0 +1,532 @@
+..   .. latex::
+..      :usepackage: footmisc
+
+:author: Bradley D. Dice
+:email: bdice@umich.edu
+:institution: Department of Physics, University of Michigan, Ann Arbor
+:institution: These authors have an equal contribution to the work presented.
+
+:author: Brandon L. Butler
+:email: butlerbr@umich.edu
+:institution: Department of Chemical Engineering, University of Michigan, Ann Arbor
+:institution: These authors have an equal contribution to the work presented.
+
+:author: Vyas Ramasubramani
+:email: vramasub@umich.edu
+:institution: Department of Chemical Engineering, University of Michigan, Ann Arbor
+
+:author: Alyssa Travitz
+:email: atravitz@umich.edu
+:institution: Macromolecular Science and Engineering Program, University of Michigan, Ann Arbor
+
+:author: Michael M. Henry
+:email: mikehenry@u.boisestate.edu
+:institution: Micron School of Materials Science and Engineering, Boise State University
+
+:author: Hardik Ojha
+:email: hojha@ch.iitr.ac.in
+:institution: Department of Chemical Engineering, Indian Institute of Technology Roorkee
+
+:author: Carl S. Adorf
+:email: TODO
+:institution: Department of Chemical Engineering, University of Michigan, Ann Arbor
+
+:author: Sharon C. Glotzer
+:email: sglotzer@umich.edu
+:institution: Department of Physics, University of Michigan, Ann Arbor
+:institution: Department of Chemical Engineering, University of Michigan, Ann Arbor
+:institution: Biointerfaces Institute, University of Michigan, Ann Arbor
+
+:bibliography: paper
+
+-------------------------------------------------------------------
+signac: Data Management and Workflows for Computational Researchers
+-------------------------------------------------------------------
+
+.. class:: abstract
+
+The signac data management framework (https://signac.io) helps researchers execute reproducible computational studies, scales workflows from laptops to supercomputers, and emphasizes portability and fast prototyping.
```

Comment:
The previous SciPy proceedings paper used `signac` in `code font` throughout the paper. In our documentation, we use **signac** in **bold font.** How should we format the name of the project in this paper?

Reply:
I would go with bold here. I can add.